python/datashaper/datashaper/workflow.py

```diff
@@ -304,55 +294,49 @@ def output(self, id: Optional[str] = None) -> Table:
     def run(
         self,
         context: Optional[Context] = None,
-        status_reporter: Optional[StatusReporter] = NoopStatusReporter(),
-        create_verb_progress_reporter: Optional[
-            Callable[[str], StatusReportHandler]
-        ] = _create_default_verb_status_reporter,
+        options: WorkflowOptions = WorkflowOptions(),

```

Comment:
In python we usually avoid this, as this variable gets created when the class is loaded and not every time the method is called, this could cause confusion or errors, a way of avoiding this is to set it to None and then inside the method ask if it is None and set it to WorkflowOptions(),